tests/admin_changelist/tests.py

```diff
@@ -64,8 +64,8 @@ def _mocked_authenticated_request(self, url, user):
 
     def test_select_related_preserved(self):
         """
-        Regression test for #10348: ChangeList.get_queryset() shouldn't
-        overwrite a custom select_related provided by ModelAdmin.get_queryset().
+        ChangeList.get_queryset() shouldn't
```

Comment:
The line lengths should be reflowed to 79 characters so there isn't a short line followed by a long line.